BuildPropertiesPlugin/README.md

```diff
@@ -0,0 +1,157 @@
+# Android build properties plugin
+**TL;DR** A Gradle plugin to add easy support of property files to your Android build.<br/>
+<br/>
+
+Sometimes you want to retrieve some information from a file that is not checked in as part of your repo for security reasons (keys, credentials, passwords, etc).
+Other times it's just you already are given a properties file and you need to make sure it ends up in your application `BuildConfig` or in some resource file.<br/>
+This plugin augments the `android` default facilities, aiming to provide a simple way to:
+- define handles to properties file in your build script (à la `signingConfig`)
+- generate string fields in your `BuildConfig` with values from a properties file
+- generate string resources with values from a properties file
+- load signing configurations from properties files avoiding to worry about passwords and keystores references in your build script
+
+## How To Build
+
+To build the latest version of the plugin, run from the root of the repo:
+```
+./gradlew build
+```
+
+The plugin at the moment is not deployed in any public maven repo, therefore your only option is to deploy it to a local one.
+A default setup with an embedded repo (pointing to `.gradle/repo`) is provided, just run the following command from the root
+ of the project:
+```
+./gradlew -Plocal uploadArchives
```

Comment:
you can still use the bintray release plugin to deploy locally, altho it is a bit more wordy than this